src/styles/main.scss

```diff
@@ -1,3 +1,37 @@
+@import "./variables";
+@import "./calendar";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day {
```

Comment:
The .calendar__day class should be inside the .calendar class to follow the BEM methodology.